helm-charts/README.md

```diff
@@ -0,0 +1,156 @@
+# How to run signals-gisib-ams app in Kubernetes using Helm
+This README provides instructions on how to deploy the `signals-gisib-ams`
+application in Kubernetes using Helm. Before deploying the application, we need
+to deploy two dependencies: PostgreSQL and RabbitMQ.
+
+However, note that deployment of these components is **optional** if you already
+have other solutions like an Azure database for PostgreSQL or another instance
+of RabbitMQ that you wish to use.
```

Comment:
This works, but I think for local testing [subcharts](https://helm.sh/docs/chart_template_guide/subcharts_and_globals/) would work even better.